commands/roles.js

```diff
@@ -0,0 +1,22 @@
+const { roleBlacklist } = require('../config.json');
+
+module.exports = {
+	name: 'roles',
+	description: 'Lists all self-assignable roles.',
+	aliases: [],
+	usage: '[roles]',
+	cooldown: 5,
+	guildOnly: true,
+	execute(message, args) {
```

Comment:
We don't need `args` here, as there aren't any arguments for this command.
```suggestion
	execute(message) {
```